Makefile

```diff
@@ -43,11 +44,18 @@ $(BUILDJS_TARGETS): $(BUILDJS_SOURCES) build.js
 
 
 MODULE_TARGETS = \
-	js/lib/id/actions.js
+	js/lib/id/actions.js \
+	js/lib/id/ui.js
 
 js/lib/id/actions.js: modules/
 	node_modules/.bin/rollup -f umd -n iD.actions modules/actions/index.js --no-strict > $@
 
+js/lib/id/ui.js: modules/ui/*
```

Comment:
I think the actions.js target above also needs proper dependencies defined, i.e. `modules/actions/*`, right?